views/tags/login.tag

```diff
@@ -0,0 +1,13 @@
+<login>
```

Comment:
@SimonLab is this the latest version of login.tag? Do double check and get back to me.